Src/LogData.c

```diff
@@ -36,316 +73,209 @@ void writeToEEPROM(uint8_t* buffer, uint16_t bufferSize, uint16_t memAddress, ui
  * @brief Reads data from the EEPROM over I2C.
  * @param receiveBuffer, pointer to the buffer to store received data.
  * @param bufferSize, size of data to be read.
- * @param timeout, time to wait for read operation before timing out in ms.
  */
-void readFromEEPROM(uint8_t* receiveBuffer, uint16_t bufferSize, uint16_t memAddress, uint16_t timeout)
+void readFromEEPROM(uint8_t* receiveBuffer, uint16_t bufferSize, uint16_t memAddress)
 {
 	HAL_I2C_Mem_Read(&hi2c1, deviceAddress<<1, memAddress, sizeof(memAddress), receiveBuffer, bufferSize, timeout);
 }
 
 /**
  * @brief Blocks the current thread until the I2C state is ready for reading/writing.
- * @param timeout Timeout duration
  */
-void checkEEPROMBlocking(uint16_t timeout)
+void checkEEPROMBlocking()
```

Comment:
Would `wait/blockUntilEEPROMReady(...)` be a better name?

Reply:
@AgamAulakh @zelongzhang Thoughts?

Reply:
I think waitUntilEEPROMReady should be good. checkEEPROM sounds like it should return a bool but it doesn't.